content/ADR-284-private-voice-chat.md

```diff
@@ -0,0 +1,362 @@
+---
+layout: adr
+adr: 284
+date: 2025-06-30
+title: Private voice chat
+authors:
+  - LautaroPetaccio
+  - pentreathm
+status: Review
+type: Standards Track
+spdx-license: CC0-1.0
+---
+
+## Abstract
+
+This specification defines the private voice chat protocol, designed to provide voice communication between two users in the Explorer, simulating a two person call.
+
+## Context
+
+After having had the voice chat for nearby users in previous versions of Decentraland, the need of having private voice conversations between users was identified as an important feature.
+
+In this new feature, a user (now on called the **caller**) is able to ring another user (from now on the **callee**), which initiates the call flow.
+The callee has the capability of rejecting this call or accepting it and the caller can cancel the process at any time.
+If the callee chooses to reject it or the caller cancels it, the ringing process will end, notifying the caller about the decision the callee made.
+Case contrary, by accepting the call, the callee and the caller will both receive tokens to connect to the real time service that will be providing the service for the call ([LiveKit](https://livekit.io/) at the time of this ADR).
+Both users can end the call at any time, which will result in the other user being disconnected from the call.
+
+The feature considers a some restrictions, with the aim to improve the user experience, that need to be enforced:
+
+- A user can't be in more than one private voice chat at a time.
+- A user can't be ringing or be ringed by another user at the time.
+- A user can't start a private voice chat with another user if at least one of them has their privacy settings set to _only friends_ and their are not friends.
+- A user can't be ringing someone or be ringed be someone if they're not connected.
+
+Following the user restrictions mentioned above, there are some restrictions for the voice chat that are required to be enforced as well:
+
+- A ringing private voice chat can't be ringing for an indeterminate amount of time, they must be expired.
+- A private voice chat where one of the users didn't connect for an expected amount of time must not exist
+- A private voice chat where one of the users abruptly disconnected for an expected amount of time must not exist
```

Comment:
Not sure I get this sentence, what does abruptly mean in this context? what should not exists is a call with a single user, right?

Reply:
You're right, this wasn't really clear. I've changed it for:
```
A private voice chat where one of the users disconnected, due to connectivity issues, for an expected amount of time must not exist
```

Reply:
lgtm